packages/apps/graph/package.json

```diff
@@ -8,28 +8,38 @@
   "author": "",
   "main": "index.js",
   "scripts": {
-    "build": "pnpm prisma:generate && heft build --clean",
+    "prebuild": "pnpm run pactjs:generate:contract:chain",
+    "build": "pnpm prebuild && pnpm prisma:generate && heft build --clean",
```

Comment:
If pnpm doesn't support pre/post commands, then we might as well just do `"build": "pnpm run pactjs:generate:contract:chain && pnpm prisma:generate && heft build --clean",` and remove the prebuild command

Reply:
prebuild removed 👍🏽